client/src/templates/Challenges/classic/editor.css

```diff
@@ -174,17 +174,21 @@ textarea.inputarea {
   }
 }
 
-.lower-jaw-icon-bar {
-  display: flex;
-  gap: 10px;
-  flex-direction: row;
-}
-.lower-jaw-icon-bar > button {
-  line-height: 0;
+.utility-bar {
+  display: grid;
+  /* 8rem is a safe default for all currently supported languages. Will need to be increased if a new language has longer words for Reset/Help. */
+  grid-template-columns: repeat(auto-fill, minmax(8rem, 1fr));
+  gap: 0.75rem;
+  align-items: center;
+  justify-content: center;
```

Comment:
The nitpick:

The grid override these values, so they don't shift, unless I have missed something.

```suggestion
```